readthedocs/search/faceted_search.py

```diff
@@ -96,53 +95,112 @@ class PageSearchBase(RTDFacetedSearch):
     }
     doc_types = [PageDocument]
     index = PageDocument._doc_type.index
-    fields = ['title^10', 'headers^5', 'content']
+
+    outer_fields = ['title^10']
+    section_fields = ['sections.title^5', 'sections.content']
```

Comment:
I was advised to remove the boosts here, as ES is pretty good at handling it themselves.

Reply:
Okay.
I was myself confused about what should be the `optimal` boosting for the fields.
But, ES is quite smart about it.